invenio_app_ils/ui/src/invenio_app_ils/common/components/Document/DocumentLanguages.js

```diff
@@ -0,0 +1,47 @@
+import {List} from 'semantic-ui-react';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+export class DocumentLanguages extends Component {
+
+  renderPrefix = () => {
+    if (this.props.prefix){
+      return this.props.prefix;
+    }
+    return null;
+  };
+
+  render() {
+    const {metadata, delimiter} = this.props;
+    return (
+      <>
+        {this.renderPrefix()}
```

Comment:
```suggestion
        {this.props.prefix}
```
Should be enough and remove `renderPrefix` completely.

Reply:
actually, `undefined` is also allowed, I will remove my comment then:
https://stackoverflow.com/a/42083262/675178